### ISSUE TITLE
[Heroic - Escape from Durnholde Keep] Captain Skarloc RP broke and don't start if player (-s) go/are invisible during RP before fight

### ISSUE THREAD

**Issue by kerricsson** [chromiecraft/chromiecraft#6079]
### What client do you play on?

enGB

### Faction

Alliance

### Content Phase:

70

### Current Behaviour

NPC: https://wowgaming.altervista.org/aowow/?npc=17862

Captain Skarloc RP broke and don't start the fight if player (-s) (=druids/rogues) go/are invisible during RP before fight. Only way to restart encounter is to aggro rest of mobs and let then kill Thrall. After his death boss encounter can be restarted and completed if rogue/druid in party will not go to invisible state.

### Expected Blizzlike Behaviour

Rogue/druid invisibility state should not break boss RP and start boss encounter.

### Source

Obvious reasons.

### Steps to reproduce the problem

1. Go to HC version of this dungeon.
2. Escort Thrall to this boss.
3. Go to invis and observe script state.

### AC rev. hash/commit

https://github.com/chromiecraft/azerothcore-wotlk/commit/4c287e2873c3464a64baafc8c93a47521bccfbc9

### Operating system

Ubuntu 20.04

### Modules

- [mod-ah-bot](https://github.com/azerothcore/mod-ah-bot)
- [mod-bg-item-reward](https://github.com/azerothcore/mod-bg-item-reward)
- [mod-BreakingNewsOverride](https://github.com/AnchyDev/BreakingNewsOverride)
- [mod-cfbg](https://github.com/azerothcore/mod-cfbg)
- [mod-chat-transmitter](https://github.com/azerothcore/mod-chat-transmitter)
- [mod-cta-switch](https://github.com/azerothcore/mod-cta-switch)
- [mod-desertion-warnings](https://github.com/azerothcore/mod-desertion-warnings)
- [mod-duel-reset](https://github.com/azerothcore/mod-duel-reset)
- [mod-eluna](https://github.com/azerothcore/mod-eluna)
- [mod-instanced-worldbosses](https://github.com/nyeriah/mod-instanced-worldbosses)
- [mod-ip-tracker](https://github.com/azerothcore/mod-ip-tracker)
- [mod-low-level-arena](https://github.com/azerothcore/mod-low-level-arena)
- [mod-low-level-rbg](https://github.com/azerothcore/mod-low-level-rbg)
- [mod-multi-client-check](https://github.com/azerothcore/mod-multi-client-check)
- [mod-progression-system](https://github.com/azerothcore/mod-progression-system)
- [mod-pvp-titles](https://github.com/azerothcore/mod-pvp-titles)
- [mod-pvpstats-announcer](https://github.com/azerothcore/mod-pvpstats-announcer)
- [mod-queue-list-cache](https://github.com/azerothcore/mod-queue-list-cache)
- [mod-rdf-expansion](https://github.com/azerothcore/mod-rdf-expansion)
- [mod-transmog](https://github.com/azerothcore/mod-transmog)
- [mod-weekend-xp](https://github.com/azerothcore/mod-weekend-xp)
- [mod-zone-difficulty](https://github.com/azerothcore/mod-zone-difficulty)
- [lua-carbon-copy](https://github.com/55Honey/Acore_CarbonCopy)
- [lua-event-scripts](https://github.com/55Honey/Acore_eventScripts)
- [lua-exchange-npc](https://github.com/55Honey/Acore_ExchangeNpc)
- [lua-level-up-reward](https://github.com/55Honey/Acore_LevelUpReward)
- [lua-recruit-a-friend](https://github.com/55Honey/Acore_RecruitAFriend)
- [lua-send-and-bind](https://github.com/55Honey/Acore_SendAndBind)
- [lua-temp-announcements](https://github.com/55Honey/Acore_TempAnnouncements)
- [lua-zonecheck](https://github.com/55Honey/acore_Zonecheck)

### Customizations

None

### Server

CC live - AzerothCore rev. 4c287e2873c3+ 2023-08-22 12:13:36 +0200 (master branch) (Unix, RelWithDebInfo, Static)

**Comment by balleny** [comment]
duplicate of https://github.com/chromiecraft/chromiecraft/issues/5972
